internal/internal_workflow_client.go

```diff
@@ -56,8 +56,8 @@ var (
 )
 
 type (
-	// workflowClient is the client for starting a workflow execution.
-	workflowClient struct {
+	// WorkflowClient is the client for starting a workflow execution.
+	WorkflowClient struct {
```

Comment:
Why do we have to make concrete implementation public?  Can we just another API on client to retrieve Options?